INSTALL.md

```diff
@@ -76,16 +76,16 @@ To get the latest features and patches as soon as they are released, or to contr
 0. Make sure you have [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) installed for version control.
 1. From the terminal, clone the repository. This will add the provided URL to the list of remotes under the name `origin`:
     ```
-    git clone https://github.com/Qiskit-Extensions/staged-primitives.git
+    git clone https://github.com/pedrorrivero/pyproject-qiskit.git
```

Comment:
Add disclaimer saying that the commands are generic and should be substituted for the appropriate URLs and package names